lib/pdk/validators/ruby/rubocop.rb

```diff
@@ -16,6 +16,10 @@ def self.cmd
         'rubocop'
       end
 
+      def self.pattern
+        ['**/*.rb']
```

Comment:
That's a good point (although our `.rubocop.yml` template in puppetlabs/pdk-module-template is explicitly excluding `Gemfile` and `Rakefile` at the moment, but we can fix that later). I'll remove the pattern and update the spec